cypress/integration/pages/liveRadio/helper.js

```diff
@@ -1,5 +1,20 @@
+import envConfig from '../../../support/config/envs';
+
 // the externalId `bbc_oromo_radio` is overriden to `bbc_afaanoromoo` in production code
 const getMappedServiceId = externalId =>
   externalId === 'bbc_oromo_radio' ? 'bbc_afaanoromoo_radio' : externalId;
 
-export default getMappedServiceId;
+export const getEmbedUrl = (body, language) => {
+  const { id, externalId } = body.content.blocks[2];
+  const mediaId = getMappedServiceId(externalId);
+
+  return [
+    envConfig.avEmbedBaseUrl,
+    'ws/av-embeds/media',
+    mediaId,
```

Comment:
```suggestion
    serviceId,
```

Could also replace line 15 with "liveradio" if preferred - it is always live radio